base/gfspclient/signer.go

```diff
@@ -544,3 +544,47 @@ func (s *GfSpClient) RejectMigrateBucket(ctx context.Context, rejectMigrateBucke
 	}
 	return resp.GetTxHash(), nil
 }
+
+func (s *GfSpClient) Deposit(ctx context.Context, deposit *virtualgrouptypes.MsgDeposit) (string, error) {
+	conn, err := s.SignerConn(ctx)
+	if err != nil {
+		log.Errorw("failed to connect to signer", "error", err)
+		return "", ErrRPCUnknownWithDetail("client failed to connect to signer, error: " + err.Error())
```

Comment:
when [pr 1257](https://github.com/bnb-chain/greenfield-storage-provider/pull/1257) is merged, returned error statement should be fixed.